src/apps/replication/lib/replica_stub.cpp

```diff
@@ -531,7 +533,8 @@ void replica_stub::on_learn_completion_notification(const group_check_response&
 
 void replica_stub::on_add_learner(const group_check_request& request)
 {
-    replica_ptr rep = get_replica(request.config.gpid, true, request.app_type.c_str());
+    // TODO(qinzuoyan): here call get_replica() with new_when_possible = ?
+    replica_ptr rep = get_replica(request.config.gpid, false, request.app_type.c_str());
```

Comment:
seems the dir will be auto-deleted now in replica::newr. In this case, false is correct. If the dir is not auto-deleted, then true is also ok.